Makefile

```diff
@@ -92,6 +92,38 @@ client/contracts/truffle/node_modules:
 client/build/bin/abigen:
 	cd client; build/env.sh go run build/ci.go install ./cmd/abigen
 
+go-generate: moq go-bindata stringer gencodec mockery ensure-notifications ensure-wallet-backend
+	go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
Move this to its own task at the end (with the other tools) and only install it if missing